compiler-core/schema.capnp

```diff
@@ -90,6 +90,11 @@ struct RecordAccessor {
   label @2 :Text;
 }
 
+struct FunctionArgument {
+  name @0 :Text;
```

Comment:
Function types don't contain names! Please remove this.

It would also be called a parameter. Things have parameters, and for those parameters they accept arguments.

Reply:
So you mean that a parameter has a name/label, while an argument doesn't? In that case, I'd reuse the `TypeValueConstructorParameter` struct?

Reply:
To be fair I wasn't happy with the approach I took 😅 . Tomorrow I'll try tackling this again.

Reply:
Neither has names at the type level, but parameters can have names locally within a definition.